### PR TITLE
Send the human readable format in ContentItem

### PR DESCRIPTION
**Pull request by tommyp**
For Policies, we want to specify if it's a Policy Area or Policy Programme on the page. This commit sends it in the ContentItem which is then passed to the title component as the context which is rendered above the title. 

Example screenshot: 
screen shot 2015-02-16 at 10 59 03

### CODE REVIEW

**Comment by jamiecobbett** [comment]
No tests?

**Comment by tommyp** [comment]
It's checking against the class name which I'm fairly confident will work. Is it necessary to introduce a test for something as basic as this?

**Comment by elliotcm** [comment]
Yeah, firstly for regression, second for clarity of intent, both most useful if the code is changed later.

File: app/presenters/content_item_presenter.rb [hunk 1]
```diff
@@ -69,10 +69,20 @@ def details
...
   end
+
+  def human_readable_finder_format
+    case policy.class.name
+    when "PolicyArea"
+      "Policy Area"
+    when "Programme"
+      "Policy Programme"
+    end
+  end
```
Comment (elliotcm) [inline app/presenters/content_item_presenter.rb]: Can do this more simply as:
```
case policy
    when PolicyArea
      "Policy Area"
    when Programme
      "Policy Programme"
    end
```

**Comment by tommyp** [comment]
Changed the class check based on @elliotcm's comment and added a test for the `ContentItemPresenter`.

File: spec/presenters/content_item_presenter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  let(:policy) {
+    PolicyArea.new(
+      name: "Tea and biscuits",
+      slug: "tea-and-biscuits",
+      description: "Tea and biscuits are popular among many people who live in the UK.",
+      content_id: "f61106bd-fb70-4003-a85a-71e5d52bea01",
+    )
+  }
+
+  let(:presenter) { ContentItemPresenter.new(policy) }
```
Comment (dsingleton) [inline spec/presenters/content_item_presenter_spec.rb]: Doesn't need changing, but you can do `described_class.new(policy)` for the same effect, which is quite nice.

**Comment by fatbusinessman** [comment]
A small style point: in general, we use sentence case for headings on GOV.UK unless they’re names of specific things (for example, the [Access to Work](https://www.gov.uk/access-to-work/overview) scheme or the [Freedom of Information Act](https://www.gov.uk/make-a-freedom-of-information-request/the-freedom-of-information-act)). More relevant to this instance, we use lower case for types of content, such as [press releases](https://www.gov.uk/government/news/pork-pies-merger-may-give-rise-to-competition-concerns) and [policy papers](https://www.gov.uk/government/publications/minimum-fish-landing-sizes).

If you want to have a poke around in more depth, there are more examples [in the style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#capitalisation).

**Comment by edds** [comment]
@fatbusinessman this has now been addressed in https://github.com/alphagov/policy-publisher/pull/15

**Pull request merged**